doc/releases/changelog-0.43.1.md

```diff
@@ -3,12 +3,14 @@
 
 <h3>Bug fixes 🐛</h3>
 
-* Fixes a bug where mid-circuit measurements were generating incomplete QASM.
+* Fixes a bug in QASM output where the `creg` declaration for mid-circuit measurements was missing 
+  a semicolon, causing parsing failures in external tools.
```

Comment:
Is the broader issue that the QASM is generally invalid without the semicolon? Or can QASM without the semicolon still run in qiskit code?